website/src/scripts/calculate-starting-point.ts

```diff
@@ -0,0 +1,63 @@
+import { dominantConsonants, passiveConsonants } from '../data/letter-hierarchy';
+import { findOrCreateOutlineObject } from './build-outline-objects';
+import allOutlines from '../data/outlines.json';
+import type { OutlineObject } from '../data/interfaces/interfaces';
+
+export const createStartingObject = (text: string, lettersObjectArray: OutlineObject[]) => {
+	// Identify dominant letter
+	const findDominantLetter = (word: string | undefined) => {
+		const dominantConsonant = word.split('').find((letter) => dominantConsonants.includes(letter));
```

Comment:
What if word is `undefined?`

```suggestion
		const dominantConsonant = word?.split('').find((letter) => dominantConsonants.includes(letter));
```